link_shortener/form_routes.py

```diff
@@ -129,14 +181,14 @@ async def update_link_save(request, user, status, link_id):
                 table.update().where(
                     table.columns['id'] == link_id
                 ).values(
+                    password=form.password.data,
```

Comment:
Do I understand correctly that if you fill in only one of these fields in the form, the others are going to be cleared after submitting the form? Is it intentional?

Reply:
This one is up to business specification. 
- If you change all fields, unfilled attributes will have empty values.
- If you keep unchanged fields, you won't be able to set empty values (and thus delete an existing password for example).

There probably is a way to do both, but all my ideas would make the edit form very cluttered.

Reply:
CMIIW but I think that if you set the original values of the fields as a _values_ of those input fields (instead of just a _placeholder_) then the user is able to both change the values, set empty values or preserve the original values, aren't they?

Reply:
Placeholder replaced by value for URL.